flocker/provision/_install.py

```diff
@@ -125,30 +54,30 @@ def task_test_homebrew(recipe):
     :return: List of commands used to install a Homebrew recipe for Flocker and
```

Comment:
These no longer return "list"s but instead `sequence`s. Though perhaps they would be better returning lists and the performers can worry about casting to `sequence`s.